src/librustdoc/html/render.rs

```diff
@@ -810,11 +810,9 @@ themePicker.onblur = handleThemeButtonsBlur;
                 if line.starts_with(&format!("\"{}\"", krate)) {
                     continue;
                 }
-                if line.ends_with(",\\") {
-                    ret.push(line[..line.len() - 2].to_string());
-                } else {
+                if let Some(head) = line.strip_suffix(",\\").or_else(|| line.strip_suffix("\\")) {
                     // Ends with "\\" (it's the case for the last added crate line)
-                    ret.push(line[..line.len() - 1].to_string());
+                    ret.push(head.to_string());
```

Comment:
re https://github.com/rust-lang/rust/pull/74048/files#r449881783

The comment above this statement confirms that the `else` case `line` ends with "\\".
cc @GuillaumeGomez who wrote this code in b4fb3069ce82f61f84a9487d17fb96389d55126a

Reply:
Normally, all lines should end with ",\\" (except the last one which ends with "\\").

Reply:
So does I reserve old behavior here?

Reply:
Yes please